### PR TITLE
Fix for PDF display and Photo input size

### PR DESCRIPTION
**Pull request by sudhacheran**
1. Utilized the 'browser-image-compression' library to compress images to a specific size limit.
2. Defined image container CSS to display image in a fixed size.
3. Updated the worker JavaScript files for the 'pdfjs-dist' library to ensure compatibility with the 'react-pdf' renderer.
4. Corrected a typo reported by Edward.

### CODE REVIEW

File: src/components/photo_input_wrapper.tsx [hunk 1]
```diff
@@ -39,44 +42,34 @@ const PhotoInputWrapper: FC<PhotoInputWrapperProps> = ({
...
+     * @throws {Error} - Throws an error if the compression process fails.
+     *
+     */
+    async function compressFile(imageBlob: Blob) {
+        /*The compressed file will have a maximum size of `maxSizeMB` 
+         and dimensions constrained by`maxWidthOrHeight`. */
+        const options = {
+            maxSizeMB: 0.2,
+            useWebWorker: true,
+            maxWidthOrHeight: 1950,
```
Comment (markborkum) [inline src/components/photo_input_wrapper.tsx:63]: Should this be `Math.max(MAX_IMAGE_DIM_WIDTH, MAX_IMAGE_DIM_HEIGHT)`?

File: src/components/photo_input_wrapper.tsx [hunk 2]
```diff
@@ -39,44 +42,34 @@ const PhotoInputWrapper: FC<PhotoInputWrapperProps> = ({
...
-                )
-            }
-
-            img.onerror = () => {
-                reject(new Error('Image loading failed'))
-            }
-
-            img.src = URL.createObjectURL(jpegBlob) // Create an object URL for the blob
-        })
+        console.log(compressedFile.size / 1024 / 1024)
```
Comment (markborkum) [inline src/components/photo_input_wrapper.tsx:71]: Please remove `console.log` statement.

Also, `a/x/x === a`, so this should be `compressedFile.size / (1024 * 1024)`.

File: src/components/photo_input_wrapper.tsx [hunk 3]
```diff
@@ -39,44 +42,34 @@ const PhotoInputWrapper: FC<PhotoInputWrapperProps> = ({
...
+     *                                         and the function may return undefined.
+     *
+     * @throws {Error} - Throws an error if the compression process fails.
+     *
+     */
+    async function compressFile(imageBlob: Blob) {
+        /*The compressed file will have a maximum size of `maxSizeMB` 
+         and dimensions constrained by`maxWidthOrHeight`. */
+        const options = {
+            maxSizeMB: 0.2,
```
Comment (markborkum) [inline src/components/photo_input_wrapper.tsx:61]: Can `0.2` be refactored to a global variable or a property?

File: src/components/photo_input_wrapper.tsx [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
 import PhotoMetadata from '../types/photo_metadata.type'
 
 import heic2any from 'heic2any'
 
 import { getMetadataFromPhoto } from '../utilities/photo_utils'
+import { size } from 'lodash'
 
-const MAX_IMAGE_DIM_WIDTH = 500
-const MAX_IMAGE_DIM_HEIGHT = 350
+const MAX_IMAGE_DIM_WIDTH = 800
```
Comment (markborkum) [inline src/components/photo_input_wrapper.tsx:15]: Can these constants be refactored to properties?

**Review by markborkum — CHANGES_REQUESTED** [review]
Minor changes requested.

File: src/components/photo_input_wrapper.tsx (hunk 3 above)
Comment (sudhacheran) [reply src/components/photo_input_wrapper.tsx]: Done

File: src/components/photo_input_wrapper.tsx (hunk 2 above)
Comment (sudhacheran) [reply src/components/photo_input_wrapper.tsx]: Thanks :-) Removed the console.log.

File: src/components/photo_input_wrapper.tsx (hunk 1 above)
Comment (sudhacheran) [reply src/components/photo_input_wrapper.tsx]: Thanks for catching this. I have updated it accordingly.

**Review by markborkum — APPROVED** [review]
Looks good to me. Thanks, @sudhacheran!

**Pull request merged**